staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/escaping.go

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
This file is missing unit tests.

Reply:
This file only contains basic string manipulation and it will be covered by escaping_test.go wondering if the unit test here is needed

Reply:
I've reorganized the tests for escaping so that there is a unit test next to `escaping.go` and a bunch of cases explaining the behavior in `validation_test.go`